packages/react-native-editor/bin/generate-pot-files.sh

```diff
@@ -114,8 +114,9 @@ function make_pot () {
   # In order to detect parse errors, we need to use the "--debug" option, otherwise "make-pot" command doesn't output errors.
   local full_command="$makepot_command $source --debug $arguments"
   if [[ -z ${DEBUG:-} ]]; then
-    # When DEBUG flag is not enabled, we only print the parse errors.
-    $full_command 2> >(grep 'Could not parse file' | sed 's/Debug[[:space:]][(]make-pot[)]/Warning/g' >&2)
```

Comment:
I decided to remove the `sed` part because it was causing the output to be printed later than expected and after other log entries. Its purpose was mainly to change `Debug (make-pot)` message to `Warning`, now that we print extra debug information, it's not too bad keeping the original value.